app/assets/css/mq.css

```diff
@@ -7,16 +7,16 @@
      */
     .menu-on #nav {
         animation: slide 0.3s ease-out;
+        background: var(--color-bg);
         box-shadow: 1px 0 9px 1px #555;
```

Comment:
does this need a var?

Reply:
Removed here https://github.com/acss-io/atomizer/pull/505/commits/68ae81e63bbde3fcc9dc07626fc153884b6c7a97